wordpress-mobile/WordPress-iOS
# Feature/4105 sharing buttons

This pr updates the data, network, and service layers to support syncing and storing the various third-party service buttons that a blog can display for the purposes of share posts. The UI for managing settings related to the sharing buttons will come in a subsequent PR. 

To test:
Confirm the app builds without errors and you are not logged out due to the new model version. 
Run the test suite and confirm the new tests pass. 

Needs review: @jleandroperez
